include/crocoddyl/core/diff-action-base.hpp

```diff
@@ -86,8 +86,8 @@ class DifferentialActionModelAbstractTpl {
   /**
    * @brief Compute the total cost value for nodes that depends only on the state
    *
-   * It updates the total cost when system acceleration as zero. This function is commonly used in the
-   * terminal nodes of an optimal control problem.
+   * It updates the total cost and the system acceleration is not updated as it is expected to be zero. This function
```

Comment:
I still don't get why the system acceleration should be "expected to be zero". Since the control is not defined in the terminal node then also the acceleration is undefined.

Reply:
Just this one: https://github.com/loco-3d/crocoddyl/pull/995#discussion_r708269942